README.md

```diff
@@ -93,6 +93,16 @@ url = 'https://raw.githubusercontent.com/pykeen/pykeen/master/src/pykeen/dataset
 df: pd.DataFrame = pystow.ensure_csv('pykeen', 'datasets', 'nations', url=url)
 ```
 
+Ensure a comma-separated tabular data file from the internet and load it for usage (requires `pip install pandas`):
+
+```python
+import pystow
+import pandas as pd
+
+url = 'https://raw.githubusercontent.com/pykeen/pykeen/master/src/pykeen/datasets/nations/test.txt'
```

Comment:
this is a tab-separated dataset... not sure this example makes sense

Reply:
Ah, right. Added a different file.